web-marketplace/src/components/atoms/ExampleProjectLink.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link as RouterLink } from 'react-router-dom';
```

Comment:
why do we need this? does not seem to be used

Reply:
It's something I extracted from a [component](https://github.com/regen-network/regen-web/blob/0502b306deff636c49237d70e0d5059499d67619/web-components/src/components/section/OnBoardingSection.tsx#L90-L97). But this part was never used. Should I remove this component or add some comment to indicate it's related to the `OnBoardingSection` component ?

Reply:
oh I see let's just drop it then